source/assets/scripts/main.js

```diff
@@ -110,6 +111,7 @@ function getTasksFromStorage() {
  * Takes in an array of tasks and for each tasks creates a
```

Comment:
In the beginning let's add a more general description: get array of tasks and add them to corresponding week day list as `<task-card>` objects